src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
```

Comment:
Redundant annotation. The @Data annotation already includes @ToString. Consider removing redundant annotations to reduce clutter.